uaclient/status.py

```diff
@@ -120,7 +120,8 @@ class UserFacingStatus(enum.Enum):
 
 MESSAGE_REFRESH_ENABLE = 'Refreshing contracts prior to enable'
 MESSAGE_REFRESH_SUCCESS = 'Refreshed Ubuntu Advantage contracts.'
-MESSAGE_REFRESH_FAILURE = 'Failure to refresh Ubuntu Advantage contracts.'
+MESSAGE_REFRESH_FAILURE = """\
+Ignoring failure to refresh Ubuntu Advantage contracts."""
```

Comment:
Why prompted this change?

Reply:
only a passing comment from andreas about maybe reformatting the text we log which agreed with my misgiving about starting a log message with 'Failure' in our logs that really is not treated as an operational failure because we ignore it